### PR TITLE
update instructions for openconnect

### PR DESCRIPTION
**Pull request by arashbm**
Cisco VPN now requires User Agent to contain the word AnyConnect to work.

https://scicomp.zulip.cs.aalto.fi/#narrow/stream/4-general/topic/VPN.20connection.20problem/near/276672

### CODE REVIEW

**Comment by tpfau** [comment]
When I try to do this, I don't get to the SSO handler, which would be needed to log in. 
How do you set up the SSO handler for openconnect?

**Comment by arashbm** [comment]
On Ubuntu 23.10 package `network-manager-openconnect-gnome` has a dependency on `libwebkit2gtk-4.0-37` and just opens up a webkit when connecting to the VPN. Seems like invoking from CLI has a [known issue](https://gitlab.com/openconnect/openconnect/-/issues/425) though. Should I just remove mentions to the CLI?

**Comment by tpfau** [comment]
I think the problem is in exactly this package. Ubuntu 22.04 `network-manager-openconnect-gnome` is still at version 1.2.6-4, which (I think) does not have the fix setting AnyConnect as client ID for cisco vpn servers, and thus fails... 
I'm currently waiting for April to upgrade and use the next lts which hopefully has solved this.

**Comment by arashbm** [comment]
At least on 1.2.10-1 the user agent still has to be set manually.

**Comment by tpfau** [comment]
But how do you set it at all for the UI version? 
In my network manager, there is no optio to set the user Agent anywhere.

**Comment by arashbm** [comment]
It seems to be there in the newer version:
image

**Comment by tpfau** [comment]
Ok, well... I'll be waiting till 24.04 then since 22.04 doesn't seem to get that upgrade. :/

**Comment by rkdarst** [comment]
Is there any reason to not merge this right now?

File: aalto/remoteaccess.rst [hunk 1]
```diff
@@ -70,8 +70,9 @@ Below is some quick reference info.
 -  Aalto windows: Start → Search → AnyConnect
 -  Personal Linux laptops: Use OpenConnect. Configuration on Ubuntu:
    Networks → Add Connection → Cisco AnyConnect compatible VPN. →
-   ``vpn.aalto.fi``. Then connect and use Aalto username/password. Or from
-   command line: ``openconnect https://vpn.aalto.fi``
+   Set Gateway to ``vpn.aalto.fi`` and User Agent to ``AnyConnect''.
+   Then connect and use Aalto username/password. Or from the command
+   line: ``openconnect https://vpn.aalto.fi --useragent=AnyConnect``
```
Comment (rkdarst) [inline aalto/remoteaccess.rst:75]:
```suggestion
   line: ``openconnect https://vpn.aalto.fi --useragent=AnyConnect``.  This requires a new enough OpenConnect, at least Ubuntu 23.10 has it.
```

**Comment by rkdarst** [comment]
I'm just merging this, it's better than before.  I'm sorry it took so long - thanks for the info!

**Pull request merged**